arrow-fx-test/src/main/kotlin/arrow/test/laws/ConcurrentLaws.kt

```diff
@@ -119,7 +119,7 @@ object ConcurrentLaws {
         Law("Concurrent Laws: RaceN arity-8 should be stack safe") { CF.race8StackSafe(iterations, EQK.liftEq(Int.eq()), ctx) },
         Law("Concurrent Laws: RaceN arity-9 should be stack safe") { CF.race9StackSafe(iterations, EQK.liftEq(Int.eq()), ctx) }
       )
-    } else emptyList()) + TimerLaws.laws(CF, CF.timer(), EQK)
```

Comment:
Why is this removed?

Reply:
Duplicated below on the public laws function